### DISCUSSION TITLE
Uptime in hh:mm:ss format

### DISCUSSION THREAD

**Discussion by Ragingfire105** [dylandoamaral/uptime-card#87] [Ideas]
I am using this card to create a heatmap of sorts for watch time on the kids TV.  I used the thresholds and aliases to get the visual behavior I would like.  Now I would like to be able to show the up time of the sensor as time instead of a percentage.  I tried playing with the templates to math out the time but, with no JavaScript experience I'm at a loss. 

![image](https://user-images.githubusercontent.com/59106079/128968873-f6fefecd-a7cc-4e62-b31c-13bb6ce494d3.png)

**Comment by dylandoamaral** [comment]
I think this is a good idea, I will implement that in September for sure after my holidays.

**Reply by Ragingfire105** [reply]
Awesome, Thank you.

**Comment by dylandoamaral** [comment]
Refere to https://github.com/dylandoamaral/uptime-card#generic-interpolations for the answer

**Comment by Ragingfire105** [comment]
This has started producing an error instead of the intended output.  Regrettably, I don't know when this started happening.

**Reply by dylandoamaral** [reply]
It is a too generic answer for me to help you !

**Reply by Ragingfire105** [reply]
Sorry,  the place I am using this feature i requested only sees use during the summer when I have to monitor our kids screen time.  Which for us started this week.  As you can see in the image instead of the time is says "error".  If I remove the option it goes back to a percentage.  I am guessing one of the functions in use has been deprecated.  
![Screenshot_20240607-152506](https://github.com/dylandoamaral/uptime-card/assets/59106079/e40cc5c1-5c2a-42a7-8825-84b79863efd4)

**Reply by Ragingfire105** [reply]
What is producing the error is the code bit listed here: https://github.com/dylandoamaral/uptime-card#generic-interpolations
